addons/point_of_sale/static/tests/generic_helpers/order_widget_util.js

```diff
@@ -98,3 +101,19 @@ export function hasTax(amount) {
         trigger: `.order-summary .tax:contains("${amount}")`,
     };
 }
+export function negateStep(step) {
+    return {
+        ...step,
+        content: `Check that: ---${step.content}--- is not true`,
+        trigger: negate(step.trigger),
+    };
+}
+export function refresh() {
+    return run(() => window.location.reload(), "refresh page");
+}
+export function elementDoesNotExist(selector) {
+    return {
+        content: `Check that element "${selector}" don't exist.`,
+        trigger: negate(selector),
+    };
+}
```

Comment:
these functions should be in a separate file. ex: `/generic_helpers/utils.js`

Reply:
It is been updated according to the above mention folder structure.
Thanks!